src/components/PageBlocks/CommentConfigurator.vue

```diff
@@ -0,0 +1,246 @@
+              :key="field.name"
+              class="field"
+            >
+              <span v-if="field.label">{{ field.label }} ({{ field.name }})</span>
+              <span v-else>{{ field.name }}</span>
+              <span class="small float-right">
+                <span v-if="field.isSystem">{{ $t('field.selector.systemField') }}</span>
+                <span v-else>{{ field.kind }}</span>
+              </span>
+            </div>
+          </div>
+        </div>
+      </b-form-group>
+
+      <b-form-group
+        horizontal
+        :label-cols="3"
+        breakpoint="md"
+        :label="$t('block.recordList.record.prefilterLabel')"
+      >
+        <b-form-textarea
+          v-model.trim="options.filter"
+          :value="true"
+          :placeholder="$t('block.recordList.record.prefilterPlaceholder')"
+        />
+        <b-form-text>
+          <i18next
+            path="block.recordList.record.prefilterFootnote"
+            tag="label"
+          >
+            <code>${recordID}</code>
+            <code>${ownerID}</code>
+            <code>${userID}</code>
+          </i18next>
+        </b-form-text>
+      </b-form-group>
+
+      <b-form-group
+        horizontal
+        :label-cols="3"
+        breakpoint="md"
+        :label="$t('block.comment.titleField.label')"
+      >
+        <b-form-select v-model="options.titleField">
+          <option value="">
+            {{ $t('general.label.none') }}
+          </option>
+          <option
+            v-for="(field, index) in selectedModuleFieldsTypeString"
+            :key="index"
+            :value="field.name"
+          >
+            {{ field.label || field.name }} ({{ field.kind }})
+          </option>
+        </b-form-select>
+        <b-form-text>{{ $t('block.comment.titleField.footnote') }}</b-form-text>
+      </b-form-group>
+
+      <b-form-group
+        horizontal
+        :label-cols="3"
+        breakpoint="md"
+        :label="$t('block.comment.contentField.label')"
+      >
+        <b-form-select v-model="options.contentField">
+          <option value="">
+            {{ $t('general.label.none') }}
+          </option>
+          <option
+            v-for="(field, index) in selectedModuleFieldsTypeString"
+            :key="index"
+            :value="field.name"
+          >
+            {{ field.label || field.name }} ({{ field.kind }})
+          </option>
+        </b-form-select>
+        <b-form-text class="text-secondary small">
+          {{ $t('block.comment.contentField.footnote') }}
+        </b-form-text>
+      </b-form-group>
+      <b-form-group
+        horizontal
+        :label-cols="3"
+        breakpoint="md"
+        :label="$t('block.comment.referenceField.label')"
+      >
+        <b-form-select v-model="options.referenceField">
+          <option value="">
+            {{ $t('general.label.none') }}
+          </option>
+          <option
+            v-for="(field, index) in selectedModuleFieldsTypeString"
```

Comment:
this does not look right. you should probably filter by field type Record, correct?